.github/workflows/pr-ci.yaml

```
@@ -0,0 +1,49 @@
+# This workflow will install dependencies, run tests for both plugin and library components
+name: CloudFormation CLI  Java Plugin Pull Request CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    env:
+      AWS_DEFAULT_REGION: us-east-1
+      AWS_REGION: us-east-1
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8]
```

linking https://github.com/aws-cloudformation/cloudformation-cli-python-plugin/issues/139 again
we should just sweep all these repos when we get Python 3.9 working:
```suggestion
        python: [3.6, 3.7, 3.8, 3.9]
```